doc/lfe_guide.txt

```diff
@@ -1002,6 +1002,14 @@ Predefined LFE functions
               (refp x) and (referencep x)
               (tuplep x)
 
+   Utility and System Information Functions
```

Comment:
Hrm, looks like the plain text that was generated didn't get formatted well. I'll take a look at the .md and try again ...